bin/lint.rb

```diff
@@ -684,21 +684,6 @@ def self.check_bolded_heading(contents)
     end
   end
 
-  def self.zenodo_api(contents)
-    find_matching_texts(contents, /(zenodo\.org\/api\/files\/)/)
-      .map do |idx, _text, selected|
-      ReviewDogEmitter.error(
-        path: @path,
-        idx: idx,
-        match_start: selected.begin(1),
-        match_end: selected.end(1) + 1,
-        replacement: nil,
-        message: 'The Zenodo.org/api URLs are not stable, you must use a URL of the format zenodo.org/record/..., apologies we cannot fix automatically.',
-        code: 'GTN:032'
-      )
-    end
-  end
-
```

Comment:
really sorry but it looks like something has gone weirdly in the merge here; we just added these lines recently in a PR, but here they're being removed and I suspect it isn't just a "github display the PR weirdly" issue, but part of the commits. Is there any chance you could extract just the files you want from this PR?

Reply:
Hi @hexylena I was just writing to you when I spot this comment. Yes I think something went wrong as before the disappearing branches (on our side) PR was linting correctly